ionoscloud/resource_dbaas_mariadb_cluster_test.go

```diff
@@ -219,6 +233,7 @@ data ` + constant.DBaaSMariaDBClusterResource + ` ` + constant.DBaaSClusterTestD
 const mariaDBBackupsDataSourceMatchClusterID = mariaDBClusterConfigBasic + `
 data ` + constant.DBaaSMariaDBBackupsDataSource + ` ` + constant.DBaasMariaDBBackupsDataSourceName + ` {
 	cluster_id = ` + constant.DBaaSMariaDBClusterResource + `.` + constant.DBaaSClusterTestResource + `.id
+    depends_on = [time_sleep.wait_30_seconds]
```

Comment:
add a comment here, so we know why it was addede

Reply:
@cristiGuranIonos done